crates/orchestrator/src/tests/jobs/mod.rs

```diff
@@ -20,3 +25,103 @@ async fn create_job_fails_job_already_exists() {
 async fn create_job_fails_works_new_job() {
     // TODO
 }
+
+use std::str::FromStr;
+
+impl FromStr for JobStatus {
```

Comment:
what do we need this and the below one for

Reply:
These `impl` help `rstest` cases to convert from `String` input to `JobType/JobStatus` output to be used as function arguments.

Reply:
can we not use something like this - https://docs.rs/rstest/latest/rstest/attr.rstest.html#local-lifetime-and-by_ref-attribute